script.js

```diff
@@ -1,169 +1 @@
-    })
-}
-
-selectShows.addEventListener('change', checkChoosenShowValue)
-
-function checkChoosenShowValue() {
-    const idShow = selectShows.options[selectShows.selectedIndex].id
-    getShows(idShow)
-    console.log(idShow)
-}
-
-
-function appendToTheSelect(objectValues) {
-    while (selectEpisodes.firstChild) {
-        selectEpisodes.firstChild.remove();
-    }
-    let option = document.createElement("option")
-    option.text = "All episods"
-    option.setAttribute("value", 'all-episods');
-    selectEpisodes.appendChild(option)
-
-    objectValues.forEach((element) => {
-        let option = document.createElement("option")
-        option.text = element.name + " " + episodeCode(element)
-        selectEpisodes.appendChild(option)
-        option.setAttribute("value", `${element.id}`);
-    })
-
-    selectEpisodes.addEventListener('change', checkChoosenValue)
-
-    function checkChoosenValue(e) {
-        // const id = select.options[select.selectedIndex].id
-        const id = e.target.value
-        console.log('id', id)
-        if (id === ("All episods")) {
-            makePageForEpisodes(objectValues)
-        } else {
-            const filteredEpisod = filterTheOption(objectValues, id)
-            makePageForEpisodes(filteredEpisod)
-        }
-    }
-}
-
-// array and string 
-//3 things for the function/ input(always a parameter)/ how many parameters and what data types / and output, whic his whatever it returns and the side effect
-function filterTheOption(arrayOfEpisods, id) {
-    return arrayOfEpisods.filter(episod => id === episod.id.toString())
-}
-
-
-
-function searchTheInput(episodsObject) {
-    function inputValue() {
-        let valueOfFilter = episodsObject.filter(episod => {
-            let lowerCaseSummary = episod.summary.toLowerCase();
-            let lowerCaseName = episod.name.toLowerCase();
-            let lowerCaseInput = inputTag.value.toLowerCase();
-            let episodeCodeowerCase = episodeCode(episod).toLowerCase()
-            if (lowerCaseSummary.indexOf(lowerCaseInput) > -1 || lowerCaseName.indexOf(lowerCaseInput) > -1 || episodeCodeowerCase.indexOf(lowerCaseInput) > -1) {
-                return true
-            } else {
-                return false
-            }
-        })
-        makePageForEpisodes(valueOfFilter)
-        displayPtagValue(valueOfFilter, episodsObject)
-
-    }
-    inputTag.addEventListener('input', inputValue)
-}
-
-function displayPtagValue(arrayToBeFiltered, wholeArray) {
-    pLabelTag.textContent = `Displaying ${arrayToBeFiltered.length}/${wholeArray.length}`
-}
-
-
-function episodeCode(episodObject) {
-    let episodSeason = episodObject.season;
-    let episodNumber = episodObject.number;
-
-    episodSeason = episodSeason < 10 ? `0${episodSeason}` : episodSeason;
-    episodNumber = episodNumber < 10 ? `0${episodNumber}` : episodNumber;
-
-    return `S${episodSeason}E${episodNumber}`;
-}
-
-function removePtags(objectEpisods) {
```

Comment:
This is a good function, my one is removing all HTML tags and yours is for <p> tags only so it is better